.github/workflows/publish.yml

```
@@ -14,7 +14,7 @@ jobs:
     - uses: actions/checkout@v2
     
     - name: Pre-requisities
-      run: sudo npm install -g vsce
+      run: sudo npm install -g vsce@1.103.1
```

Did you this working somewhere? Because I tried it here: https://github.com/michalbali256/code4z/runs/4334798573?check_suite_focus=true and it still ends with the same issue.

This worked for me: 
```suggestion
      run: npm install -g vsce
```

The action still ends with an error but it does produce an artifact.

---

In COBOL LS we have the same error, but there we used `npx` instead of `npm install`.
I didn't test it.

---

You are right. Now I think that it will be better to run vsce via `npx` like we do for `semantic-release`.